cmd/recall/main_tui.go

```diff
@@ -0,0 +1,28 @@
+//go:build tui
+
+package main
+
+import (
+	"github.com/unowned-ai/recall/pkg/tui"
+
+	"github.com/spf13/cobra"
+)
+
+var tuiCmd = &cobra.Command{
+	Use:   "tui",
+	Short: "Show terminal UI",
+	Long:  `Display an interactive terminal UI for browsing data.`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		dbConn, err := openDB()
+		if err != nil {
+			return err
+		}
+		defer dbConn.Close()
+
+		return tui.ShowTUI(dbConn)
+	},
+}
+
+func init() {
```

Comment:
The reason you use `init` is because of the build flag? Honestly, I feel as if the flags are unnecessarily making our build complex.

Is it possible to include or exclude individual lines based on build flags? If so, I would prefer if we simply do a `addCommand` in the same file where we define the CLI (this is how we handle journals I believe).

Also, can we avoid using global variables like `tuiCmd`?

Reply:
I also do not like too much of global vars, I need we need to refactor it, all CLI to be true